Apps/WebClient/src/ClientApp/app/components/navmenu/navHeader.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <b-navbar toggleable="lg" type="dark">
+  <b-navbar toggleable="md" type="dark">
```

Comment:
why this change

Reply:
makes the hamburguer menu show up only in medium size, prevents it from happening while printing the page, and it does fit in the medium view.